src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/model/sar/SARConditionFormatter.kt

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.digital.hmpps.hmppshdcapi.model.sar
+
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.conditions.ConditionMetadata
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.conditions.V1_CONDITIONS
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.conditions.V2_CONDITIONS
+
+typealias ConditionPolicies = Map<Int, List<ConditionMetadata>>
+
+object SARConditionFormatter {
+
+  val policyVersions = mapOf(1 to V1_CONDITIONS, 2 to V2_CONDITIONS)
+
+  // Map of condition to version (but only include conditions that only appear in a single policy version)
+  private val conditionToPolicyVersion =
+    policyVersions.entries
+      .flatMap { it.value }
+      .groupingBy { it.id }
+      .eachCount()
+      .filter { it.value == 1 }
+      .map { it.key to policyVersions.findVersion(it.key) }.toMap()
+
+  fun ConditionPolicies.getConditionText(conditionVersion: Int?, id: String): String = policyVersions[conditionVersion]?.find { it.id == id }?.text ?: id
+
+  fun ConditionPolicies.getConditionValues(values: Map<String, Any>): Map<String, Any>? = values.takeIf { it.isNotEmpty() }?.mapKeys { it.key.camelToSentenceCase() }
+
+  fun ConditionPolicies.getPolicyVersion(conditionIds: Collection<String>) = conditionIds.mapNotNull { conditionToPolicyVersion[it] }.distinct().singleOrNull()
```

Comment:
mechanism to guess to the policy version based on a set of condition ids - this is required for licenceVersions which don't store the additional condition id. 
There is an edge case where we wouldn't be able to guess - a couple of the conditions have ids that exist across policy versions.. if only these conditions were selected then we aren't able to tell which policy version to use.

In the indeterminate cases we just fall back to returning the raw condition id.